src/azure-cli-testsdk/azure/cli/testsdk/utilities.py

```diff
@@ -48,14 +48,8 @@ def force_progress_logging():
     az_logger.handlers[0].level = old_az_level
 
 
-def _py3_byte_to_str(byte_or_str):
-    import logging
```

Comment:
We don't need log here now?

Reply:
Why do we need log here? 🤣 This function simply converts `bytes` to `str`. We seldom care about the warning logged by test framework during a test.